eks-distro-base/scripts/install_golang

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
```

Comment:
This along with the versions in the makfile are duplicated from the builder-base build.  I am not sure i want to introduce a dep in the build process yet between these two so opting for copying for now until we see our update pattern and how much of a pain this is.  I also intend to remove the golang versions from the builder-base and switch to using these images directly, but that could be a longer term vision.